mypyc/irbuild/ll_builder.py

```diff
@@ -622,14 +622,17 @@ def compare_tagged(self, lhs: Value, rhs: Value, op: str, line: int) -> Value:
         self.goto_and_activate(out)
         return result
 
+    def unary_not(self,
+                  value: Value,
+                  line: int) -> Value:
+        return self.binary_int_op(bool_rprimitive, value, value, BinaryIntOp.NOR, line)
```

Comment:
This is problematic, since `x NOR x` is `~x` (bitwise negation), whereas we want to only touch the lowest order bit.